### PR TITLE
Create room component

### PR DESCRIPTION
**Pull request by Estaffieri**
## Description
This PR is starting to build out the Room component and experimenting with some of the navigation techniques in React Native.

## Motivation and Context
One of the main features of this project is to have rooms for users to see and revisit memories in. We also need to find a way to to change views within the ios simulator.

## How Has This Been Tested?
Viewed in ios simulator.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by brycemara** [comment]
Good start!

**Comment by brycemara** [comment]
Looking good.

File: src/Room/Room.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { StatusBar } from "expo-status-bar";
+import React from "react";
+import { StyleSheet, Text, View, Image } from "react-native";
+
+export default Room = ({ navigation }) => {
+    return (
+        <View style={styles.container}>
+            <Text>Living Room</Text>
+            <Image source={require("/assets/roomImages/livingroom.png")} style={{width: 100% , height: 100%}}></Image>
```
Comment (Codo-Baggins) [inline src/Room/Room.js:9]: This is great, I'm assuming we will be dynamically passing this image based on which Room is fetched, right?

File: src/Stack/Stack.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+                <Stack.Screen
+                    name="Room"
+                    component={App}
+                    options={{ title: 'Living Room' }}
+                />
+                <Stack.Screen name="HomePage" component={HomePage} />
+            </Stack.Navigator>
+        </NavigationContainer>
+    );
+};
```
Comment (Codo-Baggins) [inline src/Stack/Stack.js:22]: Is this navigation working correctly yet?

**Review by Codo-Baggins — APPROVED** [review]
Good job overall! I really just want to know if that navigation is working properly.

**Review by cameronRomo — APPROVED** [review]
NavigationContainer seems very similar to router, thankfully! Thanks for incorporating this functionality.

**Pull request merged**